doc/akri_connector/connector-metadata-schema.json

```diff
@@ -134,11 +134,10 @@
     }
   },
   "properties": {
-    "aioConnectorMetadataSchemaVersion": {
+    "$schema": {
       "type": "string",
-      "description": "The version of AIO connector metadata schema that this connector metadata file adheres to.",
-      "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9A-Za-z-][0-9A-Za-z-]*)(?:\\.(?:0|[1-9A-Za-z-][0-9A-Za-z-]*))*))?(?:\\+([0-9A-Za-z-]+(?:\\.[0-9A-Za-z-]+)*))?$",
-      "default": "6.0-preview"
+      "description": "The address of the AIO connector metadata schema that this connector metadata file adheres to.",
+      "default": "https://raw.githubusercontent.com/Azure/iot-operations-sdks/refs/heads/main/doc/akri_connector/connector-metadata-schema.json"
```

Comment:
This URL does not have the version of the schema. 
Which means this will resolve to the LATEST version of the metadata schema. 
We need to have per-version files for the schema. 
And maybe we should consider having both the version and the ref. With the ref being optional.

Reply:
Alright, fair point. But then why have pointers to 2 different locations? Why not the latest one in the schemastore?